[docs] Describe how to disable the 'close all tabs' popup in IE (#5757)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/run-tests.md

> Note that quarantine mode increases the test task's duration because failed tests are executed three to five times.

See Martin Fowler's [Eradicating Non-Determinism in Tests](http://martinfowler.com/articles/nonDeterminism.html) article for more information about non-deterministic tests.

## Troubleshooting

### Disable the 'Close Multiple Tabs' Popup in IE

> Internet Explorer displays a dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and pauses tests. Disable this popup before you run tests.

Reviewer: ```suggestion
Internet Explorer displays a dialog when you try to close a browser window with more than one opened tab. This dialog blocks the page and pauses tests. Disable this dialog before you run tests.
```

> Internet Explorer displays a native dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and halts tests. Disable this popup before you run tests.

Reviewer 2: Does Internet Explorer has non native dialogs? I guess, it doesn't.

 `Internet Explorer displays a dialog...`

`the page and halts tests` -> `the page and stops/freezes tests`
Use the simplest vocabulary, because a lot of our users are not native English speakers.

Author: ```suggestion
Internet Explorer displays a dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and pauses tests. Disable this popup before you run tests.
```

> To disable the popup, manually open multiple tabs in one window of Internet Explorer and close the browser. In the dialog window that appears, check `Always close all tabs` and click `Close all tabs`. IE saves this setting and the popup does not appear again.

Reviewer: ```suggestion
To disable this dialog, open multiple tabs in one Internet Explorer window and close the browser. Check `Always close all tabs` and click `Close all tabs`. IE saves this setting and the dialog does not appear again.
```
